HousingSearchListener/V1/Domain/Asset/AssetAddress.cs

```diff
@@ -0,0 +1,33 @@
+namespace HousingSearchListener.V1.Domain.Asset
+{
+    public class AssetAddress
```

Comment:
Can we use the shared nuget package instead 
https://github.com/LBHackney-IT/asset-shared